src/Symfony/Component/Clock/Resources/now.php

```diff
@@ -13,13 +13,20 @@
 
 if (!\function_exists(now::class)) {
     /**
-     * Returns the current time as a DateTimeImmutable.
-     *
-     * Note that you should prefer injecting a ClockInterface or using
-     * ClockAwareTrait when possible instead of using this function.
+     * @throws \DateMalformedStringException When the modifier is invalid
      */
-    function now(): \DateTimeImmutable
+    function now(string $modifier = null): \DateTimeImmutable
     {
-        return Clock::get()->now();
+        if (null === $modifier || 'now' === $modifier) {
+            return Clock::get()->now();
+        }
+
+        if (\PHP_VERSION_ID >= 80300) {
+            $now = Clock::get()->now()->modify($modifier);
+        } else {
+            $now = @Clock::get()->now()->modify($modifier) ?: throw new \DateMalformedStringException(error_get_last()['message'] ?? sprintf('Invalid date modifier "%s".', $modifier));
```

Comment:
When is there or isn't an error message?

Reply:
there's always one, unless a broken custom error handler is used :shrug: